web/ts/TUMLiveVjs.ts

```diff
@@ -203,70 +210,169 @@ export const skipSilence = function (options) {
  * @param lastProgress The last progress fetched from the database
  */
 export const watchProgress = function (streamID: number, lastProgress: number) {
-    const player = videojs("my-video");
-    player.ready(() => {
-        let duration;
-        let timer;
-        let iOSReady = false;
-        let intervalMillis = 10000;
-
-        // Fetch the user's video progress from the database and set the time in the player
-        player.on("loadedmetadata", () => {
-            duration = player.duration();
-            player.currentTime(lastProgress * duration);
-        });
+    for (let j = 0; j < players.length; j++) {
+        players[j].ready(() => {
+            let duration;
+            let timer;
+            let iOSReady = false;
+            let intervalMillis = 10000;
+
+            // Fetch the user's video progress from the database and set the time in the player
+            players[j].on("loadedmetadata", () => {
+                duration = players[j].duration();
+                players[j].currentTime(lastProgress * duration);
+            });
 
-        // iPhone/iPad need to set the progress again when they actually play the video. That's why loadedmetadata is
-        // not sufficient here.
-        // See https://stackoverflow.com/questions/28823567/how-to-set-currenttime-in-video-js-in-safari-for-ios.
-        if (videojs.browser.IS_IOS) {
-            player.on("canplaythrough", () => {
-                // Can be executed multiple times during playback
-                if (!iOSReady) {
-                    player.currentTime(lastProgress * duration);
-                    iOSReady = true;
-                }
+            // iPhone/iPad need to set the progress again when they actually play the video. That's why loadedmetadata is
+            // not sufficient here.
+            // See https://stackoverflow.com/questions/28823567/how-to-set-currenttime-in-video-js-in-safari-for-ios.
+            if (videojs.browser.IS_IOS) {
+                players[j].on("canplaythrough", () => {
+                    // Can be executed multiple times during playback
+                    if (!iOSReady) {
+                        players[j].currentTime(lastProgress * duration);
+                        iOSReady = true;
+                    }
+                });
+            }
+
+            const reportProgress = () => {
```

Comment:
I think the progress report does not work anymore. 🤔

Reply:
Locally the progress report still works. Why do you think it doesn't? 🧐